### PR TITLE
recipe for dukpy

### PR DESCRIPTION
**Pull request by pvanheus**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by pvanheus** [comment]
Recipe for [dukpy](https://github.com/amol-/dukpy). Listed myself as the maintainer.

**Comment by pvanheus** [comment]
For some reason I cannot check off the last 2 items on the checklist...

File: recipes/dukpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: 081eda6d60053dc6b141171ff6cfd5dfd037ab06d4ad5fbeaf01b850029cac3d
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (marcelotrevisani) [inline recipes/dukpy/meta.yaml]: Do you need the ``c`` compiler here?

File: recipes/dukpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/amol-/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 081eda6d60053dc6b141171ff6cfd5dfd037ab06d4ad5fbeaf01b850029cac3d
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/dukpy/meta.yaml]: you need to add the ``entry_points`` here
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  entry_points:
    - dukpy-install=dukpy.install:main
    - dukpy=dukpy.run:main
```

File: recipes/dukpy/meta.yaml (hunk 1 above)
Comment (pvanheus) [reply recipes/dukpy/meta.yaml]: Yes, in the `src/` folder there are C files that get compiled into the `dukpy._dukpy` extension.

**Comment by pvanheus** [comment]
Thanks for the review @marcelotrevisani!

**Pull request merged**